src/components/Auth/TodoInfo.tsx

```diff
@@ -0,0 +1,124 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { Loader } from './Loader';
+
+type Props = {
+  todo: Todo;
+  updateTODOCompleted: (todoId: number | undefined, completed: boolean) => void;
+  removeTodo: (todoId: number | undefined) => void;
+  updateTODOTitle: (todoId: number | undefined, title: string) => void;
+};
+
+export const TodoInfo: React.FC<Props> = ({
+  todo,
+  updateTODOCompleted,
+  removeTodo,
+  updateTODOTitle,
+}) => {
+  const [isVisible, setIsVisible] = useState(true);
+  const [currentTodoId, setCurrentTodoId] = useState(3);
+  const [value, setValue] = useState<string>(todo.title);
+
+  const handle = (
+    event: React.KeyboardEvent<HTMLSpanElement>,
```

Comment:
the wrong type is specified here. please check other places as well
```suggestion
    event: React.KeyboardEvent<HTMLInputElement>,
```